kafka/docker/constant-detector/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Copyright 2018 Expedia Group, Inc.
```

Comment:
Would be nice to use templates or something for these docker/{app}/* files as they are all almost identical.